src/points/PointsHome.tsx

```diff
@@ -153,7 +146,7 @@ export default function PointsHome({ route, navigation }: Props) {
               <NumberTicker
                 testID="PointsBalance"
                 value={pointsBalance}
-                disableAnimation={lastKnownPointsBalance.current === pointsBalance}
```

Comment:
I'm wondering if the difficulties we're seeing implementing the behavior to animate on first mount and on change are because we aren't reusing the shared `Animated.Value` and triggering the start in the each render phase. 🤔 

[new Animated.Value](https://github.com/valora-inc/wallet/blob/a012a8d212ad1b93ebb33b3a7c0ba05b92ed0e43/src/components/NumberTicker.tsx#L37-L45)

Reply:
yes you're right!! i struggled with exactly this yesterday, and tried a bunch of conditions for when to kick off this `Animated.timing`. the missing piece was reusing the animatedValue, thanks!! it works well now :D